### PR TITLE
31 error handling in case ntp server is unreachable

### PR DESCRIPTION
**Pull request by AnnaFeiler**
## 📑 Description

<add description if available>

## ☑ Mandatory Tasks

The following aspects have been respected by the pull request assignee and at least one reviewer:
  
- Changelog update (necessity checked and entry added or not added respectively)
  * [ ] Pull Request Assignee
  * [x] Reviewer

### CODE REVIEW

File: asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc [hunk 1]
```diff
@@ -78,18 +78,112 @@ NOTE: By using the device's clock accuracy, a consumer can decide if received da
...
+NOTE: This allows the <<vol1_spec_sdpi_p_actor_somds_consumer>> to decide if timestamps are still accurate enough for its specific use case.
+====
+****
+*REVIEWER QUESTION*:Is R1531 too challenging for legacy devices?
+
+
+.R1532
+[sdpi_requirement#r1532,sdpi_req_level=shall]
+****
+The <<term_manufacturer>> of a <<vol1_spec_sdpi_p_actor_somds_consumer>> shall consider the risk of providing the <<vol1_spec_sdpi_p_actor_somds_consumer>>'s <<term_system_function_contribution>>, if the accuracy of the device internal clock decreases due to an unreachable <<acronym_ts_service>>.
```
Comment (d-gregorczyk) [inline asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc:112]:
```suggestion
The <<term_manufacturer>> of a <<vol1_spec_sdpi_p_actor_somds_consumer>> shall consider the risk of providing the <<vol1_spec_sdpi_p_actor_somds_consumer>>'s <<term_system_function_contribution>> if the accuracy of the device internal clock decreases due to an unreachable <<acronym_ts_service>>.
```

File: asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc [hunk 2]
```diff
@@ -78,18 +78,112 @@ NOTE: By using the device's clock accuracy, a consumer can decide if received da
...
+
+****
+
+.R1533
+[sdpi_requirement#r1533,sdpi_req_level=shall]
+****
+The <<term_manufacturer>> of a <<vol1_spec_sdpi_p_actor_somds_consumer>> shall consider the risk of providing the <<vol1_spec_sdpi_p_actor_somds_consumer>>'s <<term_system_function_contribution>>, if the accuracy of the <<vol1_spec_sdpi_p_actor_somds_provider>>'s timestamps decreases.
```
Comment (d-gregorczyk) [inline asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc:119]:
```suggestion
The <<term_manufacturer>> of a <<vol1_spec_sdpi_p_actor_somds_consumer>> shall consider the risk of providing the <<vol1_spec_sdpi_p_actor_somds_consumer>>'s <<term_system_function_contribution>> if the accuracy of the <<vol1_spec_sdpi_p_actor_somds_provider>>'s clock decreases.
```

File: asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc [hunk 3]
```diff
@@ -78,18 +78,112 @@ NOTE: By using the device's clock accuracy, a consumer can decide if received da
...
+
+.Notes
+[%collapsible]
+====
+NOTE: The <<vol1_spec_sdpi_p_actor_somds_consumer>> can assume that the accuracy is at least 50 ms, if no other value is provided in the <<term_medical_data_information_base>> ClockState/@Accuracy.
```
Comment (d-gregorczyk) [inline asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc:124]:
```suggestion
NOTE: In accordance with <<r1531>>, a <<vol1_spec_sdpi_p_actor_somds_consumer>> can assume that the accuracy of a <<vol1_spec_sdpi_p_actor_somds_provider>>'s clock is at least 50ms if no other value is provided in the <<term_medical_data_information_base>> pm:ClockState/@Accuracy.
```

File: asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc [hunk 4]
```diff
@@ -78,18 +78,112 @@ NOTE: By using the device's clock accuracy, a consumer can decide if received da
...
+
+====
+****
+
+*REVIEWER QUESTION*:Do we need a requirement, for notifying the biomed in case the <<acronym_ts_service>> is no longer reachable? Or is the following logging requirement sufficient?
+
+.R1534
+[sdpi_requirement#r1534,sdpi_req_level=shall]
+****
+The <<vol1_spec_sdpi_p_actor_somds_participant>> shall create a log entry, if the <<acronym_ts_service>> cannot be reached.
```
Comment (d-gregorczyk) [inline asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc:136]:
```suggestion
If a <<vol1_spec_sdpi_p_actor_somds_participant>> cannot reach the <<acronym_ts_service>>, the <<vol1_spec_sdpi_p_actor_somds_participant>> shall create a log entry.
```

File: asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc [hunk 5]
```diff
@@ -78,18 +78,112 @@ NOTE: By using the device's clock accuracy, a consumer can decide if received da
...
 
 *And* The device will create a log entry noting inaccurate time synchronization
 
 *And* The device will periodically attempt to reconnect to the <<acronym_md_lan>> and <<acronym_ts_service>>
 
-*And* Based on a <<term_manufacturer>>'s risk management, the device may be disconnected entirely from the <<acronym_md_lan>> network.
\ No newline at end of file
+*And* Based on a <<term_manufacturer>>'s risk management, the device may be disconnected entirely from the <<acronym_md_lan>> network.
+
+NOTE: It is the <<vol1_spec_sdpi_p_actor_somds_consumer>>'s responsibility to decide if timestamps are still accurate enough to execute its <<term_system_function_contribution>>.
```
Comment (d-gregorczyk) [inline asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc:159]:
```suggestion
NOTE: It is the <<vol1_spec_sdpi_p_actor_somds_consumer>>'s responsibility to decide if timestamps are accurate enough to execute its <<term_system_function_contribution>>.
```

File: asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc [hunk 6]
```diff
@@ -78,18 +78,112 @@ NOTE: By using the device's clock accuracy, a consumer can decide if received da
...
+
+====== Safety, Effectiveness & Security Considerations and Requirements
+
+.R1540
+[sdpi_requirement#r1540,sdpi_req_level=shall]
+****
+If a <<vol1_spec_sdpi_p_actor_somds_consumer>> receives an <<term_medical_data_information_base>> containing a ClockState/@Accuracy that is no longer sufficient for at least one of its <<term_system_function_contribution>>, it shall disable all affected <<term_system_function_contribution>>s.
```
Comment (d-gregorczyk) [inline asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc:166]:
```suggestion
If a <<vol1_spec_sdpi_p_actor_somds_consumer>> receives an <<term_medical_data_information_base>> containing a pm:ClockState/@Accuracy that is no longer sufficient for at least one of its <<term_system_function_contribution>>s, the <<vol1_spec_sdpi_p_actor_somds_consumer>> shall disable all affected <<term_system_function_contribution>>s.
```

File: asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc [hunk 7]
```diff
@@ -78,18 +78,112 @@ NOTE: By using the device's clock accuracy, a consumer can decide if received da
...
+****
+
+.R1541
+[sdpi_requirement#r1541,sdpi_req_level=shall]
+****
+If a <<vol1_spec_sdpi_p_actor_somds_consumer>>'s  internal clock is no longer sufficient for at least one of its <<term_system_function_contribution>>, it shall disable all affected <<term_system_function_contribution>>s.
```
Comment (d-gregorczyk) [inline asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc:172]:
```suggestion
If a <<vol1_spec_sdpi_p_actor_somds_consumer>>'s internal clock is no longer sufficient for at least one of its <<term_system_function_contribution>>s, the <<vol1_spec_sdpi_p_actor_somds_consumer>> shall disable all affected <<term_system_function_contribution>>s.
```

File: asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc [hunk 8]
```diff
@@ -78,18 +78,112 @@ NOTE: By using the device's clock accuracy, a consumer can decide if received da
...
+****
+
+.R1542
+[sdpi_requirement#r1542,sdpi_req_level=shall]
+****
+If a <<vol1_spec_sdpi_p_actor_somds_consumer>> disables one or more <<term_system_function_contribution>>s it shall notify the user.
```
Comment (d-gregorczyk) [inline asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc:178]:
```suggestion
When a <<vol1_spec_sdpi_p_actor_somds_consumer>> disables one or more <<term_system_function_contribution>>s, the <<vol1_spec_sdpi_p_actor_somds_consumer>> shall notify the user.
```

File: asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc [hunk 9]
```diff
@@ -78,18 +78,112 @@ NOTE: By using the device's clock accuracy, a consumer can decide if received da
...
+****
+
+.R1543
+[sdpi_requirement#r1543,sdpi_req_level=shall]
+****
+If a <<vol1_spec_sdpi_p_actor_somds_consumer>> disables one or more <<term_system_function_contribution>>s it shall create a log entry, noting the disabled <<term_system_function_contribution>>s as well as the cause for disabling them.
```
Comment (d-gregorczyk) [inline asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc:184]:
```suggestion
If a <<vol1_spec_sdpi_p_actor_somds_consumer>> disables one or more <<term_system_function_contribution>>s, the <<vol1_spec_sdpi_p_actor_somds_consumer>> shall create a log entry, noting the disabled <<term_system_function_contribution>>s as well as the cause for disabling them.
```

File: asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc [hunk 10]
```diff
@@ -78,18 +78,112 @@ NOTE: By using the device's clock accuracy, a consumer can decide if received da
...
+[%collapsible]
+====
+NOTE: It is likely that a <<vol1_spec_sdpi_p_actor_somds_participant>> needs multiple attempts to connect to a TS service a few times during the day. The system needs to be stable against these kind of short term interruptions.
+====
+****
+
+.R1531
+[sdpi_requirement#r1531,sdpi_req_level=shall]
+****
+If a <<vol1_spec_sdpi_p_actor_somds_provider>> cannot reach the <<acronym_ts_service>> and can no longer guarantee a 50ms accuracy, it shall provide ClockState/@Accuracy.
```
[the remark this answers is not in the document]
Comment (AnnaFeiler) [reply asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc:98]: 50 ms is what we wrote in the ntp server configuration as this is sufficient for even the most demanding use cases. The idea was that consumers can rely on this accuracy, if everything works as expected. As soon as that changes, they need to decide for themselves and consequently need all information required for a decision.

**Review by d-gregorczyk — APPROVED** [review]

**Pull request merged**